Extension/src/LanguageServer/configurations.ts

```diff
@@ -523,8 +523,9 @@ export class CppProperties {
         let result: string[] = [];
         if (paths) {
             paths = this.resolveDefaults(paths, defaultValue);
+            let delimiter: string = (this.ConfigurationNames && !(this.ConfigurationNames[0].trim().toLowerCase() === "win32")) ? ":" : ";";
```

Comment:
You don't want to make this decision based on the configuration name.  You want to make it based on the `process.platform`